### ISSUE TITLE
Extra space in specific-version install commands

### ISSUE THREAD

**Issue by gabelton** [aws/copilot-cli#5740]
On [this page](https://aws.github.io/copilot-cli/docs/getting-started/install/) in the docs, it gives an example command for installing a specific version of copilot:

`curl -Lo copilot https://github.com/aws/copilot-cli/releases/download/v0.6.0/copilot-darwin && chmod +x copilot && sudo mv copilot /usr/local/bin/copilot &&  copilot --help`

When I ran this it gave unhelpful output:

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: Failed to open the file copilot: Is a directory
  0 55.9M    0 11015    0     0  12255      0  1:19:48 --:--:--  1:19:48 32880
curl: (23) Failure writing output to destination
```

I tried a few different things before I realised that there was just extra whitespace between `&&  copilot`. It works as expected when that's removed

**Comment by dannyrandall** [comment]
Good catch! 😊

**Comment by huanjani** [comment]
Thanks again for your sharp eye.
🎉 The fix is now released in v1.33.3: https://github.com/aws/copilot-cli/releases/tag/v1.33.3
